### PR TITLE
W-14094688-mule-dx-server-issue-duke

### PR DESCRIPTION
**Pull request by dukesphere**

### CODE REVIEW

File: modules/ROOT/pages/troubleshoot-loading-issues.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+include::partial$acb-ide-logos.adoc[tags="both-ides"]
+
+[[mule-dx]]
+== Mule DX API Component was not installed
+
+If you receive the error *Mule DX API Component was not installed*, wait for background Anypoint Code Builder processes to complete. 
+
+This issue typically occurs when you run *Design an API* and is often accompanied by the notification *Extracting Mule runtime: This operation may take a while...*.
+
+You can monitor the progress of backend processes:
```
Comment (dukesphere) [inline modules/ROOT/pages/troubleshoot-loading-issues.adoc]:
```suggestion
To monitor the progress of backend processes:
```

File: modules/ROOT/pages/troubleshoot-loading-issues.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+[[mule-dx]]
+== Mule DX API Component was not installed
+
+If you receive the error *Mule DX API Component was not installed*, wait for background Anypoint Code Builder processes to complete. 
+
+This issue typically occurs when you run *Design an API* and is often accompanied by the notification *Extracting Mule runtime: This operation may take a while...*.
+
+To monitor the progress of backend processes:
+
+. Press Cmd+Shift+ (Mac) or Ctrl+Shift+u (Windows) to open Output panel.
```
Comment (dukesphere) [inline modules/ROOT/pages/troubleshoot-loading-issues.adoc]:
```suggestion
. Open the Output panel by pressing Cmd+Shift+ (Mac) or Ctrl+Shift+u (Windows).
```

File: modules/ROOT/pages/troubleshooting.adoc [hunk 3]
```diff
@@ -10,5 +10,7 @@ Troubleshoot errors and warnings you receive when using Anypoint Code Builder.
 
 * xref:troubleshoot-app-loading-errors.adoc[]
 * xref:troubleshoot-mule-runtime-errors.adoc[]
+* xref:troubleshoot-loading-issues.adoc[]
```
Comment (dukesphere) [inline modules/ROOT/pages/troubleshooting.adoc]:
```suggestion
```

File: modules/ROOT/pages/troubleshooting.adoc [hunk 4]
```diff
@@ -11,4 +11,5 @@ Troubleshoot errors and warnings you receive when using Anypoint Code Builder.
 * xref:troubleshoot-app-loading-errors.adoc[]
```
Comment (dukesphere) [inline modules/ROOT/pages/troubleshooting.adoc]:
```suggestion
* xref:troubleshoot-loading-errors.adoc[]
```

**Review by greobasco — APPROVED** [review]

File: modules/ROOT/pages/troubleshoot-loading-errors.adoc [hunk 5]
```diff
@@ -39,7 +39,15 @@ The root is the top-level folder that Anypoint Code Builder generates when it cr
 
 See also, xref:upload-a-project.adoc[].
 
+[[mule-dx]]
+== Mule DX API Component was not installed
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/troubleshoot-loading-errors.adoc]: According to our style guide, the titles are capitalized. 
This title should be "Mule Dx API Component Was Not Installed", right?
Another option would be to make a three-row table (one row per error).

**Review by Crispy-Salesforce — APPROVED** [review]

File: modules/ROOT/pages/troubleshoot-loading-errors.adoc (hunk 5 above)
Comment (dukesphere) [reply modules/ROOT/pages/troubleshoot-loading-errors.adoc]: Yes, however, that's the case of the error message. ¯\_(ツ)_/¯ . I guess I should capitalize anyway.

Comment (dukesphere) [inline modules/ROOT/pages/troubleshoot-loading-errors.adoc]:
```suggestion
== Mule DX API Component Was Not Installed
```

**Pull request merged**